examples/Gordo-Workflow-High-Level.ipynb

```diff
@@ -41,90 +41,57 @@
-    "          kind: feedforward_hourglass\n",
+    "      information: Some sweet information about the model\n",
+    "    model:\n",
+    "      gordo_components.model.anomaly.diff.DiffBasedAnomalyDetector:\n",
+    "        base_estimator:\n",
+    "          sklearn.pipeline.Pipeline:\n",
+    "            steps:\n",
+    "            - sklearn.decomposition.pca.PCA\n",
+    "            - sklearn.multioutput.MultiOutputRegressor:\n",
+    "                estimator: sklearn.linear_model.base.LinearRegression\n",
+    "    name: crazy-sweet-name\n",
     "\"\"\""
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Simulate how Gordo extract required information from a config file\n",
-    "\n",
-    "##### Note:\n",
-    "This is _not_ exactly how it's actually done. We use some resources available in `gordo-infrastructure` which is not available from `gordo-components`"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Load into a normal dict\n",
-    "config = yaml.load(config, Loader=yaml.BaseLoader)\n",
-    "\n",
-    "# Model configuration\n",
-    "model_config = config['model']\n",
-    "\n",
-    "# In this case, we only build a model for a single machine\n",
-    "machine_config = config['machines'][0]\n",
-    "\n",
-    "# TODO: This is the ugliest portion, as we normally use resources [`Machine`] found in `gordo-infrastructure`\n",
-    "data_config  = {\n",
-    "    \"type\": \"TimeSeriesDataset\",  # We want to use `DataLakeBackedDataset` for data acquisition\n",
-    "    \"from_ts\": isoparse(machine_config['dataset']['train_start_date']),\n",
-    "    \"to_ts\": isoparse(machine_config['dataset']['train_end_date']),\n",
-    "    \"tag_list\": machine_config['dataset']['tags'],\n",
-    "    \"data_provider\": DataLakeProvider(storename=\"dataplatformdlsprod\", interactive=True),\n",
-    "}"
-   ]
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "### Build model from data and model configs\n",
     "\n",
-    "This also optionally takes and will return metadata `dict` updated with various model building events"
+    "`gordo_components.builder.local_build` is a generator for building each machine in your string config, much in the same actual deployment on k8s works. \n",
+    "\n",
+    "Here we'll simply cast it as a list and get the first element, which is a `Tuple[BaseEstimator, dict]` of your model and metadata"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "pipe, metadata = build_model(\n",
-    "    name=config['model'],\n",
-    "    model_config=model_config, \n",
-    "    data_config=data_config,\n",
-    "    metadata=machine_config['metadata']\n",
-    ")"
+    "pipe, metadata = list(local_build(config))[0]"
```

Comment:
Perhaps it is clearer what `local_build` is returning by using `next(local_build(config))`?